### DISCUSSION TITLE
will ceedling v1.0.0 support Unity parametrization with preprocessor?

### DISCUSSION THREAD

**Discussion by LuisAfonso95** [ThrowTheSwitch/Ceedling#970] [General]
In the latest chagelog it says "Temporarily removed preprocessor support for Unity’s parameterized test case macros TEST_CASE() and TEST_RANGE()" will this be added back in the future release or is it intended to be working in the upcoming release?

Honestly I am not even sure that's a problem, I got to experiment with the new configuration options for the preprocessor but I am curious

**Comment by mvandervoord** [comment]
This will not be part of v 1.0. We'll be restoring this functionality down the road, but it's outside the scope of this release.

**Reply by LuisAfonso95** [reply]
Thank you!

I think this won't affect me, still need to do some testing

**Comment by Daimonion1980** [comment]
Thank you for saying that this feature will be restored. Actually our tests heavily depend's on parametrization and before updating vrom 0.31 to v1.x we need this feature working.

Is there actually any progress for reactivating this?
